qemu/tests/qemu_guest_agent.py

```diff
@@ -4160,6 +4160,22 @@ def _process_qga_resource():
             qga_memory = int(qga_resources[-2]) / 1024
             return (int(qga_handles), int(qga_memory))
 
+        def _compare_peak_value(peak_list):
+            """
+            Compare peak value of high/low peak to provide basis.
+
+            :param peak_list: the list that save peak value.
+            :return value: the count number of expected situation
+            """
+
+            i = 1
+            value = 0
+            for i in range(len(peak_list)):
+                if peak_list[i] < peak_list[i-1]:
```

Comment:
When `i == 0`, `peak_list[i-1]` is `peak_list[-1]`, I suppose that is not what you want.
`i = 1` is useless, you might try this:
```
>>> i = 1;
>>> for i in range(3):
...     print(i)
... 
0
1
2
```